mediapipe/tasks/ios/test/vision/gesture_recognizer/MPPGestureRecognizerTests.m

```diff
@@ -0,0 +1,280 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "mediapipe/tasks/ios/common/sources/MPPCommon.h"
+#import "mediapipe/tasks/ios/test/vision/gesture_recognizer/utils/sources/MPPGestureRecognizerResult+ProtobufHelpers.h"
+#import "mediapipe/tasks/ios/test/vision/utils/sources/MPPImage+TestUtils.h"
+#import "mediapipe/tasks/ios/vision/gesture_recognizer/sources/MPPGestureRecognizer.h"
+
+static NSDictionary *const kGestureRecognizerBundleAssetFile =
```

Comment:
My last reviewer asked me to specify the generics for all theses files. You should see the changes soon in the FaceLandmarker that is going to GitHub right now.

Reply:
Used generics for now. Might update these to use a class `FileInfo`  in future, when I move file path methods to helpers.